src/code/PreRender.c

```diff
@@ -768,7 +768,9 @@ void PreRender_DivotFilter(PreRender* this) {
  * Applies the Video Interface anti-aliasing filter and (optionally) the divot filter to `this->fbufSave` using
  * `this->cvgSave`
  */
-void PreRender_ApplyFilters(PreRender* this) {
+#ifdef VANILLA_PAUSE_DELAY
+
+void PreRender_Calc(PreRender* this) {
```

Comment:
did you renamed the function on accident?

Reply:
that was the result of rebasing the branch